GOVERNANCE.md

```diff
@@ -171,6 +171,22 @@ CommComm Membership Checklist:
 - [ ] Add person to the invited list in the templates for the [process that generates meeting issues](https://github.com/nodejs/create-node-meeting-artifacts) 
 ```
 
+### Offboarding
+
+When CommComm members need to be off-boarded, we should attempt to off-board them in a timely
+manner.
+
+```
+CommComm Offboarding Checklist: 
+
+- [ ] Remove the person from CommComm GitHub Team
+- [ ] Submit PR to remove the person from the [CommComm email alias](https://github.com/nodejs/email/blob/master/iojs.org/aliases.json)
+- README: 
+  - [ ] If leaving on good terms, submit PR to move the individual to Emeriti the [CommComm README.md](https://github.com/nodejs/community-committee/#community-committee-emeriti)
+  - [ ] If not leaving on good terms, submit PR to remove the person from [CommComm README.md](https://github.com/nodejs/community-committee/#community-committee-members)
```

Comment:
I can understand why this is important. But this just doesn't feel good? Any possible way if we can avoid adding this to the doc?
Not blocker.

Reply:
Process like this – especially around hard interpersonal things – should be documented IMO. It ensures that we have a protocol when things _don't_ go well, which is the most important time to have documentation.

Reply:
#defensivegovernance 😉 

Better to document the process for bad times in the good times.